docs/tools/point2dem.rst

```diff
@@ -397,6 +397,10 @@ Command-line options for point2dem
     files, if those files contain Easting and Northing fields. If
     not specified, ``--t_srs`` will be used.
 
+--input-is-projected
+    Treat the input coordinates as already in the projected coordinate
+    system, avoiding the need to convert to geodetic coordinates.
```

Comment:
This should probably say: "avoiding the need to convert from ECEF to projected coordinates".

Reply:
Or maybe it can be shortened to: "Avoiding the need to convert the points from ECEF."

Reply:
Sounds good.  I was referring to the point in the code where it gets converted to geodetic and back though I guess that is a hidden implementation detail.